examples/docking-protein-protein/docking-protein-protein.cfg

```diff
@@ -31,29 +31,29 @@ nhise = 1
 hise_1 = 15
 
 [rigidbody]
-ambig_fname = 'data/e2a-hpr_air.tbl'
+ambig_fname = "data/e2a-hpr_air.tbl"
```

Comment:
Now is the perfect time to enforce `./` as in `./data` for paths if you wish. Is an idea, not a necessity,